optimum/exporters/neuron/__main__.py

```diff
@@ -418,6 +418,70 @@ def _get_submodels_and_neuron_configs_for_encoder_decoder(
     return models_and_neuron_configs, output_model_names
 
 
+def load_models_and_neuron_configs(
```

Comment:
nit: I feel it's a bit tedious to create a new function with that many parameters just to perform a small task around `get_submodels_and_neuron_configs`

Reply:
It's a function that I separate out so that I can reuse while checking the cache with hashed configs (we might be able to get rid of it one day when we are able to hash something more deterministic like the hlo...)